usl_pipeline/tests/formats/test_geotiff_loader.py

```diff
@@ -0,0 +1,35 @@
+import unittest
+from usl_pipeline.formats import geotiff_loader
```

Comment:
per
https://peps.python.org/pep-0008/#imports

```
Imports should be grouped in the following order:

Standard library imports.
Related third party imports.
Local application/library specific imports.
```

so the usl_pipeline import should come last. We'd also prefer to import modules to keep the namespaces intact, so` import rasterio` rather than `from rasterio import CRS`. So I'd have here

```
import unittest

import rasterio

from usl_pipeline.formats import geotiff_loader
```

Reply:
Done